docsite/rst/playbooks_checkmode.rst

```diff
@@ -38,7 +38,25 @@ Example::
 
 As a reminder, a task with a `when` clause evaluated to false, will
 still be skipped even if it has a `always_run` clause evaluated to
-true.
+true. 
+
```

Comment:
version added or 'starting in 2.1' is needed so people don't try using it on older versions